pkg/lns/chirpstack.go

```diff
@@ -283,10 +303,14 @@ func (c *ChirpstackClient) GetDevices() []*types.Device {
 
 				go func(devEui string) {
 					defer sem.Release(1)
-					dev := c.GetDevice(devEui)
+					dev, err := c.GetDevice(devEui)
+					if err != nil {
+						log.Printf("Cant get device %s: %s", devEui, err)
```

Comment:
for our usecase i agree a restart of the pod should be fine. however for the fine people running this in Docker (or even on hardware or w/e) we might need to come up with something better. 

maybe fail the entier sync so it can be retried at a later moment ?